### DISCUSSION TITLE
Initial temperature in DFN

### DISCUSSION THREAD

**Discussion by xyyrly** [pybamm-team/PyBaMM#3112] [Q&A] [answered]
Dear Team,

Does DFN model without thermal submodels use initial temperature to calculate interfacial current density? Does Pybamm allow to change the single parameter value of "initial temperature" to a spatial temperature vector?

Thank you!

**Comment by brosaplanella** [comment, marked as answer]
The isothermal model takes the ambient temperature as the battery temperature, which can only be defined as a function of time (not space). Initial temperature can only be a constant, so I guess the option to enforce a temperature distribution is not available. Not sure how hard this would be to add, but you can open an issue to discuss this.

**Reply by xyyrly** [reply]
Thank you!
